### ISSUE TITLE
Scheduler documentation blocks subtly wrong

### ISSUE THREAD

**Issue by DomHudson** [huggingface/transformers#1712]
## Summary
Hi, I noticed that in `optimization.py` the way many of the schedulers describe the learning rate is slightly wrong.
For example, `WarmupLinearSchedule` says
```
        Linearly increases learning rate from 0 to 1 over `warmup_steps` training steps.
        Linearly decreases learning rate from 1. to 0. over remaining `t_total - warmup_steps` steps.
```
It actually multiplies the learning rate (set in the optimizer) by this amount. I understand that this is probably implied, but I think the documentation could be a bit clearer on this aspect!

Many thanks,
Dom

**Comment by thomwolf** [comment]
Indeed, happy to welcome a PR on that if you want to fix this

**Comment by DomHudson** [comment]
Closed by https://github.com/huggingface/transformers/pull/1737
